### ISSUE TITLE
Got : Multiple commands produce when add <resource-file src"..." />

### ISSUE THREAD

**Issue by ThunderBirdsX3** [apache/cordova-ios#510]
I got and error `Multiple commands produce` when add this code.

```
<platform name="ios">
  <resource-file src="GoogleService-Info.plist" />
</platform>
```

When look at directory it have file look like.
```
- Resources
  |- Resources
  |   |- GoogleService-Info.plist
  |- GoogleService-Info.plist
```

I must to manual remove it in `Buld Phases`

# Info

> Ionic:
> 
>    ionic (Ionic CLI)  : 4.10.1 (/usr/local/lib/node_modules/ionic)
>    Ionic Framework    : ionic-angular 3.9.2
>    @ionic/app-scripts : 3.2.1
> 
> Cordova:
> 
>    cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
>    Cordova Platforms     : ios 4.5.5
>    Cordova Plugins       : no whitelisted plugins (6 plugins total)
> 
> System:
> 
>    Android SDK Tools : 26.1.1 (/Users/ThunderBirdsX3/Library/Android/sdk)
>    NodeJS            : v11.6.0 (/usr/local/Cellar/node/11.6.0/bin/node)
>    npm               : 6.7.0
>    OS                : macOS Mojave
>    Xcode             : Xcode 10.1 Build version 10B61
>

**Comment by dpogue** [comment]
I'm not 100% sure, but I believe this is fixed in the next major version via https://github.com/apache/cordova-ios/pull/373

**Comment by dpogue** [comment]
This should be fixed in cordova-ios@5.0.0, which is now available.

**Comment by usmaem** [comment]
Cordova iOS 5.1.1 -> not fixed or regression
